llvm/lib/Target/AMDGPU/Utils/AMDGPUAsanInstrumentation.cpp

```diff
@@ -121,7 +121,8 @@ static Instruction *generateCrashCode(Module &M, IRBuilder<> &IRB,
   SmallString<128> AsanErrorCallbackString;
   raw_svector_ostream AsanErrorCallbackOS(AsanErrorCallbackString);
   AsanErrorCallbackOS << kAsanReportErrorTemplate << TypeStr
-                      << llvm::itostr(1ULL << AccessSizeIndex) << EndingStr;
+                      << llvm::format("%d", 1ULL << AccessSizeIndex)
```

Comment:
Sorry, I meant just a raw << (1ULL << AccessSizeIndex) should work, you don't need a literal ::format call